lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -651,23 +762,38 @@ LogicalResult CppEmitter::emitAssignPrefix(Operation &op) {
   return success();
 }
 
+LogicalResult CppEmitter::emitLabel(Block &block) {
+  if (!hasBlockLabel(block)) {
+    return block.getParentOp()->emitError("Label for block not found.");
+  }
+  os << getOrCreateName(block) << ":\n";
+  return success();
+}
+
 static LogicalResult printOperation(CppEmitter &emitter, Operation &op) {
-  if (auto callOp = dyn_cast<mlir::CallOp>(op))
-    return printCallOp(emitter, callOp);
+  // emitc ops
   if (auto callOp = dyn_cast<emitc::CallOp>(op))
     return printCallOp(emitter, callOp);
+  if (auto constOp = dyn_cast<emitc::ConstOp>(op))
+    return printConstantOp(emitter, constOp);
```

Comment:
Do we want to use the order in the IR definition or do we want to sort alphabetically?